config/kubernetes-client/org.yaml

```diff
@@ -5,6 +5,7 @@ has_organization_projects: true
 has_repository_projects: true
 members_can_create_repositories: false
 admins:
+- brendandburns
```

Comment:
Only the GitHub admin team can be in the `admins` list: https://github.com/kubernetes/community/tree/master/github-management#github-administration-team

To get admin permissions across repos in the kubernetes-client org, can you create teams for each repo like below? :grimacing: 

```yaml
  csharp-admins:
    description: admin access to csharp
    members:
    - brendandburns
    - krabhishek8260
    privacy: closed
```

I'll then grant admin privileges to these teams.

Reply:
I took the liberty to go ahead with this and create the `go-admins` and `go-base-admins` teams (#618) which were the only repositories not belonging to a team (yet).